mission.md

```diff
@@ -0,0 +1,332 @@
+including poverty reduction, cancer research, homelessness,
+fair trade and education.
+
+One of the things that became clear to us
+while we were working for various charities
+and for-profit businesses
+was that in _most_ cases time was not used effectively
+and communication channels were poor.
+Even though mission statements were clear,
+_translating_ those words into the day-to-day actions
+that most _effectively_ achieved the mission
+was severely lacking in most cases.
+
+
+
+## _However ..._
+
+Even though our **`start-here`** repo has been read and "_starred_" by _many_ people:
+
+![start-here-stars-sept-2019](https://user-images.githubusercontent.com/194400/65373907-8b706d80-dc7b-11e9-8f6b-d844b9a975c8.png)
+
+(_evidence that there is **some clarity** in our **purpose**_)
+
+We still feel that we are doing a _poor_ job of _communicating_ the **`Mission`**
+to the extent that people have _asked_ us to clarify it.[<sup>2</sup>](#notes)
+
+Our objective with defining a _single_ clear **`Mission`** for the organisation is to help _everyone_ in the community/team _understand exactly_ what we are trying to achieve and how much _effort_ and _perseverance_ it will require along the way.
+
+
+
+# _What_ is a `Mission` (Statement)?
+
+
+
++ https://en.wikipedia.org/wiki/Mission_statement [6 min read]
++ https://www.entrepreneur.com/encyclopedia/mission-statement [2 min read]
+
+## Examples
+
+### Unicef
+
+"_advocate for the protection of children's rights,
+help meet their basic needs and expand their opportunities
+to reach their full potential_". (21) ~ https://www.unicef.org/about/who/index_mission.html
+
+![unicef-mission-statement](https://user-images.githubusercontent.com/194400/65377978-0e102180-dcaa-11e9-8bc3-7766251aa9a2.png)
+
+### charity:water
+
+"_charity:water is a nonprofit organization bringing clean and safe drinking water to people in developing countries._" (16) ~ https://my.charitywater.org/about/mission
+
+![charity:water-mission-statement](https://user-images.githubusercontent.com/194400/65378158-c0e17f00-dcac-11e9-9d27-ce4876a745e4.png)
+
+
+### Tesla
+
+"_Tesla’s mission is to accelerate the world’s transition to sustainable energy._"
+
+![tesla-mission-statement](https://user-images.githubusercontent.com/194400/65378132-752ed580-dcac-11e9-9cad-6d5619ad91f8.png)
+
+
+<hr />
+
+If you more find good examples of mission statements,
+please add them!
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat-square)](https://github.com/dwyl/start-here/issues)
+
+<hr />
+
+
+# What is _@dwyl's_ `Mission`?
+
+### "_Empower people to maximise their personal effectiveness, creativity and happiness._"
+
+Our App and Community should help people
+to discover meaningful creative work.
+We should proactively share all our learning and knowledge
+and encourage everyone else to do the same
+so that we can accelerate progress
+and build a better future together.
```

Comment:
🙌  Improve yourself, work with others and the world will be better as a default.
Start small.